asdf/config.py

```diff
@@ -0,0 +1,81 @@
+from .util import NotSet
+from .providers import get_registered_converter_providers, create_converters
+from .converter import ConverterIndex
+
+
+__all__ = ["get_config", "configure"]
+
+
+class AsdfConfig:
+    def __init__(
```

Comment:
Docstrings with examples needed (example for the last function in file).

Reply:
Added docstrings